cirq/google/api/v2/program.proto

```diff
@@ -215,8 +215,8 @@ message CircuitOperation {
   // The index of the circuit in the top-level constant table.
   int32 circuit_constant_index = 1;
 
-  // Specifier for repetitions of the circuit, including number of reps and
-  // repetition IDs.
+  // Specifier for repetitions of the circuit, which contains either a number
+  // of repetitions or a list of repetition IDs.
   RepetitionSpecification repetition_specification = 2;
 
   // Map of qubits in the circuit to the qubits used in this operation.
```

Comment:
Perhaps could be a bit more clear here, something like "Map from qubits in the inner circuit (referenced by circuit_constant_index) to the outer circuit which contains this operation." I'd suggest similar wording for `measurement_key_map` and `arg_map` below.

Reply:
Done.